File: GameUpdate/GameUpdate/EditorCore/CommandSystem/CommandDownloadManager/CommandDownloadManagerCancel.cs
using System;
using System.Collections.Generic;

public class CommandDownloadManagerCancel : Command
{
	public override void init()
	{
		base.init();
	}
	public override void execute()
	{
		mEditorCore.sendEvent(CORE_EVENT_TYPE.CANCEL_UPDATE);
	}
	public override string showDebugInfo()
	{
		return base.showDebugInfo();
	}
}

File: SyncTool/EditorCore/CommandSystem/CommandDownloadManager/CommandDownloadManagerCancel.cpp
#include "CommandHeader.h"
#include "DownloadManager.h"
#include "EditorCore.h"

void CommandDownloadManagerCancel::execute()
{
	DownloadManager* downloadManager = static_cast<DownloadManager*>(mReceiver);
	mEditorCore->sendEvent(CET_CANCEL_UPDATE);
}

std::string CommandDownloadManagerCancel::showDebugInfo()
{
	return TOSTRING(CommandDownloadManagerCancel);
}